src/checks/graphiteThreshold.check.js

```diff
@@ -33,16 +33,27 @@ class GraphiteThresholdCheck extends Check {
 
 	generateUrl(metric, period) {
 		const urlBase = this.graphiteBaseUrl + `format=json&from=-${period}&target=`;
-		const functionName = this.direction === 'above' ? 'maximumAbove' : 'minimumBelow';
-		return urlBase + `${functionName}(${metric},${this.threshold})`;
+		const functionName = this.direction === 'above' ? 'maxSeries' : 'minSeries';
+		return urlBase + `${functionName}(${metric})`;
+	}
+
+	withinThreshold(value) {
+		return this.direction === 'above' ?
+			value[0] <= this.threshold :
+			value[0] >= this.threshold;
 	}
 
 	tick(){
 
 		return fetch(this.sampleUrl)
 			.then(fetchres.json)
 			.then(sample => {
-				const ok = sample.length === 0;
+				const datapoints = sample[0].datapoints;
+
+				const filteredDatapoints = datapoints.filter(this.withinThreshold.bind(this));
```

Comment:
am I misreading it? you're filtering based on being within the threshold. so shouldn't you be checking `filteredDatapoints.length === datapoints.length` for okay-ness? might be simpler to just filter by _outside_ threshold

but then, how are the tests passing?